README.md

```diff
@@ -19,6 +19,52 @@ You can use `stack build` to build `baseDAO` executable.
 
 Run `stack exec -- baseDAO --help` to see available commands.
 
+### Deploying a contract
+
+You can dump the entire contract code into a `BaseDao.tz` file using the following command:
+
+```sh
+stack exec baseDAO -- print -n baseDAO
+```
+
+For now this can only produce a contract with empty proposal metadata and the simplest possible configuration.
```

Comment:
Perhaps we should also consider the possibility to print GameDAO now that it's merged?